### ISSUE TITLE
Mutation causing issue with new Github constructor call

### ISSUE THREAD

**Issue by gyandeeps** [octokit/octokit.js#573]
Version: 0.9.3

Code
```js
var Github = require("github");

var g1 = new Github(); 
var g2 = new Github(); // ERROR -  Cannot read property 'acceptTree' of undefined
```

Error:

```sh
TypeError: Cannot read property 'acceptTree' of undefined
    at new module.exports (~\node_modules\github\lib\index.js:96:41)
```

Issue started happening after https://github.com/mikedeboer/node-github/pull/545 was merged. As nodejs caches all the requires and code performs delete on [line 148](https://github.com/mikedeboer/node-github/blob/90d3a14805f83ca9a7725362577bf52f5d31505a/lib/index.js#L139-L148) which is mutated.

**Comment by kaizensoze** [comment]
Yeah this is a deal breaker. Suggested fix? I think I'm going to revert the merge for now since, while it's using a require in index.js, generate.js and generateTypes.js still use `JSON.parse(fs.readFileSync(routesPath, "utf8"));`.

**Comment by kaizensoze** [comment]
Went ahead and reverted it for now (f1da560022eaab8e5ff02fb37d8e85c839135ee1). Pushing a 9.3.1 release.

**Comment by kaizensoze** [comment]
Going to create a new ticket for the JSON.parse -> require issue.

Resolved by f1da560022eaab8e5ff02fb37d8e85c839135ee1.
